admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -1991,7 +1976,7 @@ var priceCalculation = (function() {
       var impactPriceTI = this.getImpactTIInputValue(obj);
       var impactPriceTE = this.computePriceTaxExcluded(impactPriceTI);
 
-      this.updateImpactTEInput(impactPriceTE, obj);
+      this.getImpactTEInput(obj).val(impactPriceTE);
```

Comment:
Careful, you are changing the behaviour here, as the removed `updateImpactTEInput` function used to update both fields tax excluded and included (and recompute tax included at the same time)

Reply:
The whole behaviour is quite touchy and prone to many errors To avoid any risk I would have kept the `update*` methods, and you can simply remove the `change` trigger code I guess

Also you need to be careful because not only the combination form is updated, the combination list also needs to be updated and it's not handled in this file. The list update is handled here https://github.com/PrestaShop/PrestaShop/blob/339f1c8fad3f960cf381ed48c72192ea22c67081/admin-dev/themes/default/js/bundle/product/product-combinations.js#L153

It relies on the `change` event so it might break this feature, if you only wanted to remove the `change` trigger for "clean code" the DON'T 😅. Because this whole code is a mess and a slight change can break many unexpected things

As long as your bug is fixed you should try to minimize the modifications as much as possible. This code is bound to be removed once the product page is redone in v8 anyway, so now is not the time to do overkill changes unless you plan on releasing a 1.7.9 next week.. ^^